allennlp/commands/configure.py

```diff
@@ -1,58 +0,0 @@
-"""
```

Comment:
This command needs usage information in the other repo.  Also, the other repo needs some CI to be set up.

Reply:
Yep, i'll add it.